tests/scalability/json_generator.py

```diff
@@ -0,0 +1,273 @@
+import random
+import rstr
+from faker import Faker
+from jsonschema import RefResolver
+from typing import Union
+
+
+class JsonDataGenerator(object):
+    """Generate a random json document based on the provided schema."""
```

Comment:
As usual, s/json/JSON here and in all other prose.

Reply:
Thought I got them all. Fixed